experimental/traffic-portal/.gitignore

```diff
@@ -12,7 +12,6 @@
 
 # compiled output
 /dist
-/.angular
```

Comment:
I don't think this should have been changed. As of right now, we don't want anything that's in `.angular`, not just the `cache` subdirectory.

Reply:
I wanted to match the behavior of a newly generated angular project and it doesn't ignore `.angular`. That having been said I don't mind adding it back since as you said, it doesn't currently do anything.

Reply:
I can't fathom why they'd add it like that. `.angular` looks like a directory meant for use by angular, and configuration used by its users is already outside of there so I have no idea why anyone wouldn't just want to ignore the whole thing.